src/features/ui/WrappedDropdownMenu/WrappedDropdownMenu.tsx

```diff
@@ -28,25 +27,21 @@ export const WrappedDropdownMenu: FC<WrappedDropdownMenuProps> = ({
 	hasError,
 	helperText,
 }) => {
-	const trigger = (
-		<Input
-			type="text"
-			placeholder={placeholder}
-			value={value}
-			onChange={() => {}}
-			error={hasError}
-			helperText={helperText}
-			endEnhancer={<IconChevronDown />}
-		/>
-	);
-
 	return (
 		<InputWrapper
 			className={classNames(styles.Container, className)}
 			label={label}
 			info={info}
 		>
-			<DropdownMenu trigger={trigger} items={items} />
+			<SelectInput
+				items={items}
+				className={className}
+				value={value}
+				label=""
```

Comment:
Could the label for `SelectInput` be used here rather than adding `label` to the wrapper?

Reply:
Have tried but we need the info tooltip in the label unfortunately. Could change in the future if the `label` type in `SelectInputProps` is updated to a ReactNode/JSX.Element.

![Screenshot 2023-02-26 at 4 22 07 PM](https://user-images.githubusercontent.com/13953984/221390645-223117f2-6eb1-4d97-82a2-1c79150d35c0.png)
![Screenshot 2023-02-26 at 4 24 13 PM](https://user-images.githubusercontent.com/13953984/221390660-aeb81840-f05d-40ff-844f-6a2978f87f83.png)